rust-symcrypt/Cargo.toml

```diff
@@ -1,5 +1,5 @@
 [package]
-name = "symcrypt"
+name = "rust-symcrypt"

```

Comment:
I originally had it named just "symcrypt" but I added the SymCrypt github as a subrepo so there was a conflict, I named the new crate "rust-symcrypt" because its what came up in my mind first, I will think about a different name here.

Reply:
One option is to leave the repo name as rust-symcrypt but publish it to crates.io as symcrypt.

Reply:
Okay I will do that

Reply:
You can set a custom path for the submodule so that it doesn't conflict with the name you want to use. Long-term, we want to move to vcpkg anyway, so I think that's the right solution.